packages/faceted-search/src/components/Badges/BadgeCheckboxes/BadgeCheckboxesForm.component.js

```diff
@@ -55,13 +48,17 @@ const createCheckboxEntity = value => checkbox => {
 	};
 };
 
-const getCheckboxes = (checkboxes, value, filterValue) =>
-	checkboxes
-		.filter(checkbox => checkbox.label.toLowerCase().includes(filterValue))
+const getCheckboxes = (checkboxes, value, filterValue) => {
+	const formatFilterValue = filterValue.trim().toLowerCase();
+
+	return checkboxes
+		.filter(checkbox => checkbox.label.toLowerCase().includes(formatFilterValue))
```

Comment:
label is still present?

Reply:
Yes, why it would'nt ? It comes from `checkboxValues` prop

Reply:
Dunno, just to be sure it will not fail :)
```suggestion
		.filter(checkbox => get(checkbox.label, '').toLocaleLowerCase().includes(formatFilterValue))
```

Reply:
ok i didn't understand, you're right it's more safe, i change that